skynet/auth/jwt.py

```diff
@@ -37,7 +59,7 @@ async def authorize(jwt_incoming: str) -> dict:
     try:
         public_key = await get_public_key(kid)
     except Exception:
-        raise HTTPException(status_code=401, detail=f'Failed to retrieve public key. {kid}')
+        raise HTTPException(status_code=401, detail=f'Failed to retrieve public key {kid}')
```

Comment:
Ah I see. We could throw above with that description since that is where it originates, and here reuse the exception message.